### ISSUE TITLE
[Crash] Breaking a Garbage Pile Crashes the Game

### ISSUE THREAD

**Issue by KaleThyself** [Heaveness/Rebirth-of-the-Night-BPG-Edition--Future-Updates-#5]
**Describe the bug**

After placing a Garbage Pile from the Rats mod, breaking it drops a missing texture item called "Plastic Waste." This then crashes the world and freezes up the instance immediately after being picked up. 

**To Reproduce**  
1. Place a garbage pile. 
2. Break it to drop the item. 
3. Pick up the item. 

**Expected behavior**

I expect this plastic waste item to not exist. 

**Screenshots**

N/A

**Info:**
 - Modpack Version: Public Alpha 1.85, Singleplayer

**Additional context**
N/A

**Comment by KaleThyself** [comment]
Here's the crashlog report: 
[crash-2022-01-25_21.24.34-server.txt](https://github.com/Heaveness/Rebirth-of-the-Night-BPG-Edition--Future-Updates-/files/7939147/crash-2022-01-25_21.24.34-server.txt)

**Comment by Heaveness** [comment]
Yeah this is caused by an error when picking up that item. I will look into removing that item in Beta 2.0 until further notice. Thanks for letting me know.
